Microsoft.Marketplace.SaaS/2018-08-31/saasapi.v2.json

```diff
@@ -943,6 +943,9 @@
               "endDate": {
                 "type": "string",
                 "format": "date-time"
+              },
+              "termUnit": {
```

Comment:
Couldn't this be an enum with `P1M` and `P1Y`.

Reply:
Yeah, I just noticed that the definition is inconsistent with the other `termUnit` definitions, fixed.

Reply:
Thank you, already updated !